### DISCUSSION TITLE
Replacement file selection dialog still doesn't accept .icns files

### DISCUSSION THREAD

**Discussion by joaofrgomes** [elrumo/macOS_Big_Sur_icons_replacements#1446] [Help with the website]
Hi! I'm one of those old-time users with a large library of icons, many of which were uploaded back when the site “accepted” .icns files but just discarded most of its internal components and completed the hosted .icns file by downscaling the 512x512@2x.png component (which, as we discussed in the old forums, was less than ideal, as some icons require simplification and customization of certain details, which are otherwise lost at smaller sizes).

I have since uploaded a few .icns files which were properly imported with the new backend, and would also like to update the older entries without having to delete them and re-upload them, so that I may maintain the download statistics; yet, the file selection dialog that opens when clicking “Replace” doesn't allow me to select .icns files, only .png ones. Seeing how the website already accepts those files for new entries, it should be a simple fix, no?

Thank you for your attention!

**Comment by SoCuul** [comment]
@elrumo Do you think you could implement this?
